File: Customer APIs/hot.backend.all/hot.api.backend/Hot.Api.Backend/Models/EcocashFundingRequest.vb
Imports System.ComponentModel.DataAnnotations
Public Class EcocashFundingRequest
    <Required>
    Public Property Amount As Decimal
    <Required>
    <StringLength(12)>
    Public Property TargetMobile As String

    Public Property OnBehalfOf As String = "Comm Shop"

    Public Property Account As Integer = 1

End Class

File: NetworkProviders/Hot-API-Client/Hot.API.Client/Models/EcoCash/EcocashFundingRequest.cs
using Hot.API.Client.Ecocash.Domain.Enums;
using System;
using System.Collections.Generic;
using System.Text;



namespace Hot.API.Client.Models
{
    public class EcocashFundingRequest
    {
        public decimal Amount { get; set; }
        public string TargetMobile { get; set; }
        public string OnBehalfOf { get; set; } = "Comm Shop";
        public EcocashAccounts Account { get; set; } = EcocashAccounts.Default;

    }
}

